readme.txt

```diff
@@ -103,6 +103,10 @@ Track changes to posts when using the block editor.
 
 == Changelog ==
 
+= 3.10.1 - October 9, 2023 =
+
+- Fix: Fixed the dynamic callback locating routine by removing the hyphen as a valid function name character.
+

```

Comment:
<!-- simple tasks that must be done -->
**🧹 chore:** As above, this will be done as part of `4.0.1` release.

```suggestion
```